pumpkin/src/command/commands/cmd_seed.rs

```diff
@@ -0,0 +1,48 @@
+use crate::command::tree_builder::require;
+use crate::command::{
+    args::ConsumedArgs, tree::CommandTree, CommandError, CommandExecutor, CommandSender,
+};
+use crate::entity::player::PermissionLvl;
+use async_trait::async_trait;
+use pumpkin_config::BASIC_CONFIG;
+use pumpkin_core::text::click::ClickEvent;
+use pumpkin_core::text::hover::HoverEvent;
+use pumpkin_core::text::{color::NamedColor, TextComponent};
+use std::borrow::Cow;
+
+const NAMES: [&str; 1] = ["seed"];
+
+const DESCRIPTION: &str = "Displays the world seed.";
+
+struct PumpkinExecutor;
+
+#[async_trait]
+impl CommandExecutor for PumpkinExecutor {
+    async fn execute<'a>(
+        &self,
+        sender: &mut CommandSender<'a>,
+        _server: &crate::server::Server,
+        _args: &ConsumedArgs<'a>,
+    ) -> Result<(), CommandError> {
+        let seed = &BASIC_CONFIG.seed;
```

Comment:
That's only the seed from the Config, Which can be empty. If the seed is empty will will generate a random one later and store it in the `world`.

Reply:
It's not in the World, it's in GeneratorInit which I can't really access.
IG ill make this into a draft until world gen is ready and has a seed.

Reply:
You should now be able to get the seed in the level. See https://github.com/Snowiiii/Pumpkin/commit/b5a8fee243c6a4e57238bf863b57f716453e3c12